File: ios/Mobile Client/Mobile Client/entities/ScanType.swift
//
//  ScanType.swift
//  Mobile Client
//
//  Created by Eric Ruck on 2/20/18.
//  Copyright © 2018 AuditPro. All rights reserved.
//

import Foundation

class ScanType {
	static let SCANNED = ScanType(id: 1, name: "Scanned")
	static let MANUAL  = ScanType(id: 2, name: "Manual")
	static let Types   = [SCANNED, MANUAL]

	init(id: Int, name: String) {
		self.id = id
		self.name = name
	}
	
	let id: Int
	let name: String
}

File: android/mobileclient/app/src/main/java/com/auditpro/mobile_client/entities/ScanType.java
/*
  AuditPRO Mobile Client Android
  Copyright 2018 AuditPRO All Rights Reserved
 */
package com.auditpro.mobile_client.entities;


/**
 * Provides legal scan types.
 * @author Eric Ruck
 */
public class ScanType {
	public static final ScanType SCANNED = new ScanType(1, "Scanned");
	public static final ScanType MANUAL = new ScanType(2, "Manual");
	public static final ScanType[] Types = { SCANNED, MANUAL };

	private ScanType(int id, String name) {
		this.id = id;
		this.name = name;
	}

	public int getId() {
		return id;
	}

	public String getName() {
		return name;
	}

	private int id;
	private String name;
}
